components/InnbyttereContent/styles.module.css

```
@@ -0,0 +1,63 @@
+.header {
+    text-align: center;
+  }
+  
+  .header > h1,
+  .header > div {
+    margin: 2rem 0;
+  }
+  
+  .header > p {
+    margin: 1rem auto;
+    max-width: 50em;
+    line-height: 1.7;
+  }
+  
+ .infoMeetingSection {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 20px;
+    background-color: var(--red-1);
+    color: white;
+    box-shadow: 0 0 10px -5px var(--red-1);
+  }
+  .infoMeetingSection > h2 {
+    margin: 1rem 0;
+  }
+
+  .icon {
+    font-size: xx-large;
+  }
+
+  .info {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 40px;
+    padding: 20px;
+  }
+
+
+  .infoLeft {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 8px;
+  }
+
+  .infoRight {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    gap: 8px;
+
+  }
+
+  .flexRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
```

On mobile the cards don't wrap to a new line, with this they are displayed under each other when the screen is too small
```suggestion
  justify-content: center;
  flex-wrap: wrap
```